### DISCUSSION TITLE
test networks

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#1940] [Q&A]
![IMG_5044](https://github.com/Cyfrin/foundry-full-course-cu/assets/173145181/2af57e11-691d-40dc-a39a-97a6f60ca733)
my test net are locked how can i open these and now what can i do please tell me

**Comment by EngrPips** [comment]
That is how it appears for me, too, but I can still switch to and use them. So I will say ignore that.

<img width="933" alt="Screenshot 2024-07-03 at 15 43 52" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/95760918/76376840-60db-49c8-b216-ead88ea1cc72">
